NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -96,11 +96,18 @@ public extension NeuroID {
      or `startSession` in the `NIDAdvancedDevice.swift` file.
      
      Marked as `@objc` because this method can be called with reflection if the ADV library is not installed.
-     Because of the reflection we use an array with a boolean instead of just boolean
+     Because of the reflection we use an array with a boolean instead of just boolean. Log the shouldCapture flag
+     in a LOG event (isAdvancedDevice setting: <true/false>.
      */
     @objc internal static func captureAdvancedDevice(
         _ shouldCapture: [Bool] = [NeuroID.isAdvancedDevice]
     ) {
+        let logEvent = NIDEvent(type: .log)
+        logEvent.m = "shouldCapture setting: \(shouldCapture)"
+        logEvent.level = "INFO"
```

Comment:
the log level is not set as an enum, but I see that its "info" and "ERROR" in this repo. Should we change it all to uppercase?

Reply:
or maybe that's just me. Nvm, I'll make it all uppercase.

Reply:
that's a good idea, we should make it an enum for consistency

Reply:
will make INFO upper case.